src/components/common/DashboardSidebar.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import DashboardSidebarStyled from './DashboardSidebarStyled';
+import codeClanLogo from '../assets/image/codeClanLogoWhite.png';
+import { ReactComponent as HomeIcon } from '../assets/svgs/dashboard/cli_home.svg';
+import { ReactComponent as UserGroup } from '../assets/svgs/dashboard/cli_group.svg';
+import { ReactComponent as Settings } from '../assets/svgs/dashboard/cli_settings.svg';
+import { ReactComponent as Bookmark } from '../assets/svgs/dashboard/cli_bookmark.svg';
+import { ReactComponent as Message } from '../assets/svgs/dashboard/cli_message.svg';
+import { ReactComponent as Avatar } from '../assets/svgs/dashboard/cli_avatar.svg';
+
+function DashboardSidebar({ showSidebar }) {
+  console.log(showSidebar);
+
+  return (
+    <DashboardSidebarStyled showSidebar={showSidebar}>
+      <nav class="col-2 sidebar">
+        <ul class="nav flex-column">
+          <li class="nav-item mb-5 mt-3">
+            <a class="active" href="#">
```

Comment:
Codacy found an issue: [The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5731928)